extensions/flyway/deployment/src/main/java/io/quarkus/flyway/FlywayProcessor.java

```diff
@@ -117,10 +108,6 @@ ServiceStartBuildItem configureRuntimeProperties(FlywayRecorder recorder,
 
     /**
      * Collects the configured migration locations for the default and all named DataSources.
-     * <p>
-     * A {@link LinkedHashSet} is used to avoid duplications.
```

Comment:
Dunno, this doesn't seem useless to me :)

Reply:
Well, I don't think we should add a comment every time we choose a data structure. Otherwise, we will have a loooot of comments.

If you use a Set, that's usually to avoid duplications :). I think just using it is enough documentation.